api/resources_portal/models/shipping_information.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+
+class ShippingInformation(models.Model):
+    class Meta:
+        db_table = "shipping_information"
+        get_latest_by = "created_at"
+
+    objects = models.Manager()
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    needs_shipping_address = models.BooleanField(default=False)
+    sharer_pays = models.BooleanField(default=False)
+
+    ups_code_accepted = models.BooleanField(default=False)
```

Comment:
We should probably use the same semantics for `[adjective]_[noun]` or `[noun]_[adjective]`
So either of the following:
* `needs_shipping_address` and `accepts_ups_code`
* `shipping_address_needed` and `ups_code_accepted`

Reply:
Personally, I like `[verb]_[noun]` for truthy values and `[noun]_[adjective]` for relationships and `[verb]_[noun]_[preposition]` for non-truthy vals

Reply:
I like `accepts_ups_code` more but I'm not sure I agree with your general scheme. For example, `created_at` and `updated_at` are `[verb]_[preposition]` and I can't think of a `[verb]_[noun]` that would be more clear than `sharer_pays`. (Although `sharer_pays_for_shipping` might be more clear.)

Reply:
Yeah i agree, but I think we could also describe `sharer_pays` as `requires_payment` since requirements are for the requester